SEED-platform/seed
# Apply GHG and GHGI unit versions to canonical columns

#### Any background context you want to provide?
GHG and GHGI units were selctable un column settings and mappings, however displayed values did not reflect these settings. 

#### What's this PR do?
Applies unit conversion for canonical GHG and GHGI columns during the backend's retrieval from the db.

`NOTE` This does not work for extra data columns. This is a known issue and needs further discussion.

#### How should this be manually tested?
Upload a file with canonical columns "Total GHG Emissions" and "Total GHG Emissions Intensity" (or map to those canonical columns) then use the org settings to change the display units. The values on the inventory list and details should be updated.

#### What are the relevant tickets?
#2605

#### Screenshots (if appropriate)

settings
![Screenshot 2024-01-05 at 1 29 39 PM](https://github.com/SEED-platform/seed/assets/58446472/bfce9916-5e01-4666-8748-cb462b48b577)



GHG - Mt/year & GHGI - Mt/ft2/year
![Screenshot 2024-01-05 at 1 29 04 PM](https://github.com/SEED-platform/seed/assets/58446472/b8d8b778-2591-4bcb-ad03-f576757206bc)
GHG - kg/year & GHGI - Mt/m2/year
![Screenshot 2024-01-05 at 1 48 38 PM](https://github.com/SEED-platform/seed/assets/58446472/9b90607b-60a9-4a0f-91f5-637f2ec56360)

GHG - kg/year & GHGI - kg/ft2/year
![Screenshot 2024-01-05 at 1 30 03 PM](https://github.com/SEED-platform/seed/assets/58446472/1c162260-e6f6-4cb7-ad57-26e0acf7483b)
GHG - kg/year & GHGI - kg/m2/year
![Screenshot 2024-01-05 at 1 43 23 PM](https://github.com/SEED-platform/seed/assets/58446472/daacc44c-7f44-4e0e-8a96-857bec60ca84)
